src/irep2/irep2_utils.h

```diff
@@ -1,6 +1,7 @@
 #ifndef UTIL_IREP2_UTILS_H_
 #define UTIL_IREP2_UTILS_H_
 
+#include "irep2/irep2.h"
```

Comment:
```suggestion
#include <rep2/irep2.h>
```

Can we use angle bracket here? so it looks consistent.